src/index.js

```diff
@@ -37,23 +39,26 @@ export default function loader(content) {
     const relativePath = relativeUrl && `${path.dirname(relativeUrl)}/`;
     // eslint-disable-next-line no-bitwise
     if (~relativePath.indexOf('../')) {
-      outputPath = path.posix.join(outputPath, relativePath, url);
+      outputPath = path.posix.join(outputPath, relativePath, originURL);
     } else {
-      outputPath = relativePath + url;
+      outputPath = relativePath + originURL;
```

Comment:
`path.join()` (In general) ? Things like e.g `{ outputPath: 'dist' }`  => `dist263454765.svg` if one misses the `/` atm